tools/sv-report

```diff
@@ -271,6 +271,7 @@ def collect_logs(runner_name):
             "type", "mode", "timeout", "user_time", "system_time", "ram_usage",
             "tool_success"
         ]
+        optional_test_tags = ["should_fail_because"]
```

Comment:
In my opinion `should_fail_because` should be part of `test_tags` because we already have optional tags like `top_module` in there and I don't see any benefit of separating only `should_fail_because` like this.

Reply:
Yeah it was added in the case of missing parameter but we are giving it a default value so there is no need for it